MvvmCross-iOSSupport/MvvmCross.iOS.Support/Views/MvxBaseTabBarViewController.cs

```diff
@@ -52,8 +52,10 @@ public virtual void ShowTabView(UIViewController viewController, bool wrapInNavi
 			ViewControllers = currentTabs.ToArray();
 		}
 
-		protected virtual void SetTitleAndTabBarItem(UIViewController viewController, string title, string iconName)
+		protected virtual void SetTitleAndTabBarItem(UIViewController viewController, string title, string iconName, string tabAccessibilityIdentifier)
 		{
+			viewController.View.AccessibilityIdentifier = tabAccessibilityIdentifier;
```

Comment:
Yes, I probably should also move that line to be called directly from `ShowTabView`